ads/inabox/inabox-host-babel.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Entry point into AMP inabox for compilation with babel.
+// Just loads amp-inabox.js and Babel's helpers.
+
+import '../../third_party/babel/custom-babel-helpers';
```

Comment:
Its a long story, but there is no need for the separate `-babel` file and you can just inline this into the host.